<?xml version="1.0"?>
<launch>

    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_se" clear_params="true">
        <rosparam command="load" file="$(find navigation)/config/localization_node/default.yaml" />
        <remap from="odometry/filtered" to="/odom"/>
    </node>

</launch>
